lib/killbill_client/models/combo_transaction.rb

```diff
@@ -23,14 +23,23 @@ def credit(user = nil, reason = nil, comment = nil, options = {}, refresh_option
       private
 
       def combo_payment(user, reason, comment, options, refresh_options = nil)
-        created_transaction = self.class.post "#{Payment::KILLBILL_API_PAYMENTS_PREFIX}/combo",
-                                              to_json,
-                                              {},
-                                              {
-                                                  :user => user,
-                                                  :reason => reason,
-                                                  :comment => comment,
-                                              }.merge(options)
+        begin
+          created_transaction = self.class.post "#{Payment::KILLBILL_API_PAYMENTS_PREFIX}/combo",
+                                                to_json,
+                                                {},
+                                                {
+                                                    :user => user,
+                                                    :reason => reason,
+                                                    :comment => comment,
+                                                }.merge(options)
+        rescue KillBillClient::API::ResponseError => error
+          response = error.response
+          if response.header['location']
```

Comment:
I don't think so. These types of errors are raised here:
https://github.com/andrenpaes/killbill-client-ruby/blob/0304d3cfb1909ec850d13329075bd1d108f27b62/lib/killbill_client/api/net_http_adapter.rb#L201

In this point, the response object should always be present